plugins/support/lib/nokul/support/rest_client.rb

```diff
@@ -81,14 +83,16 @@ def code
           http_response.code.to_i
         end
 
+        def decode
+          body && JSON.parse(body, symbolize_names: true)
```

Comment:
Aşağıdaki review'e göre sadece `200` kabul edeceksek `body` nil **gelemez**.  Peki neden daha önce bu durum oldu (ve `body || ''` durumuyla geçersiz JSON üretildi ki şimdi üretilmeyecek)?  Probleme öncelikle Xokul tarafında mutlaka bakmalıyız.  Ekstra emniyet için (standartlara uygun çalışmayan upstream sendromu) "panic" üreten bir kod yazabiliriz ama bence yeri burası değil `success?`.  Şöyle bir şey:

```ruby
def success?
  http_response.kind_of?(NetHTTPOK).tap do |success|
    Rollbar.log('error', 'Nil body received at 200 status') if success && body.nil?
  end
end
```

Sonuç olarak burası basitçe:

```ruby
def decode
  JSON.parse(body, symbolize_names: true)
end
```